rpcs3/Emu/Io/PadHandler.cpp

```diff
@@ -734,8 +744,14 @@ void PadHandlerBase::process()
 			if (last_connection_status[i])
 			{
 				input_log.error("%s device %d disconnected", m_type, i);
+
 				pad->m_port_status &= ~CELL_PAD_STATUS_CONNECTED;
 				pad->m_port_status |= CELL_PAD_STATUS_ASSIGN_CHANGES;
+
+				std::array<s32, CELL_MAX_PADS> status = fill_array(-1);
+				status[i] = 0 & CELL_PAD_STATUS_CONNECTED;
```

Comment:
Why & CELL_PAD_STATUS_CONNECTED ?
That's just confusing.
Just say 0 or CELL_PAD_STATUS_DISCONNECTED